readyapi4j-core/src/main/java/com/smartbear/readyapi4j/result/AbstractTestStepResult.java

```diff
@@ -10,6 +10,10 @@
 import java.util.Collections;
 import java.util.List;
 
+/**
+ * Utility class for TestStepResultReport based TestStepResult implementations
```

Comment:
I don't think we should call it utility class. Generally a utility class is with static methods and without state, but this is an abstract class.